tests/test_integration.py

```diff
@@ -352,3 +355,41 @@ def test_queryset_iterator_queries(self):
                 set([ad3.pk, self.ad1.pk, self.ad2.pk])
             )
 
+    def test_default_document_id(self):
+        obj_id = 12458
+        article_slug = "some-article"
+        article = Article(
+            id=obj_id,
+            slug=article_slug,
+        )
+
+        # saving should create two documents (in the two indices): one with the 
+        # Django object's id as the ES doc _id, and the other with the slug 
+        # as the ES _id
+        article.save()
+
+        # assert that the document's id is the id of the Django object
+        try:
```

Comment:
I think it is better to check the slug as well so we can be confirm that the object is same as that.
Something like following.
```suggestion
        es_obj = ArticleDocument.get(id=obj_id)
        self.assertEqual(es_obj._id, article.id)
        self.assertEqual(es_obj.slug, article.slug)
```